app/controllers/feature_controller.rb

```diff
@@ -0,0 +1,45 @@
+class FeatureController < ApplicationController
+  before_action :validate_feature!
+
+  # Toggles boolean for session based feature flags
+  def toggle
```

Comment:
Sure. Ticket!

The built in dashboard may be worth implementing instead of rebuilding it ourselves though. I'm thinking of something simple like making the dashboard protected by a query_param with a shared secret we link from our non-public project documentation to allow project staff to enable / disable / check status on their session for all feature toggles. Thoughts?